Documentation/operations/upgrade.rst

```diff
@@ -337,6 +337,8 @@ communicating via the proxy must reconnect to re-establish connections.
 * Support for ``metallb-bgp``, deprecated since 1.14, has been removed.
 * Layer 7 policy support for Cassandra and Memcached have been deprecated and
   their getting started guides have been removed.
+* The check for connectivity to the Kubernetes apiserver has been removed from the cilium-agent liveness probe.
+  This can be turned back on by setting the helm option ``livenessProbe.requireK8sConnectivity`` to ``true``.
```

Comment:
I'm a bit unsure whether to change this default in a bugfix release, seems a bit surprising. The least we could do is put this under a new subheading for 1.17.3 upgrade notes.